### DISCUSSION TITLE
mantine

### DISCUSSION THREAD

**Discussion by diegodesouzacap-rx** [mantinedev/mantine#6575] [Q&A] [answered]
Trying to expand on this question https://github.com/orgs/mantinedev/discussions/1112

**Comment by rtivital** [comment, marked as answer]
1. https://mantine.dev/styles/css-variables/#css-variables-resolver
2. https://mantine.dev/theming/colors/#colors-variant-resolver

**Comment by diegodesouzacap-rx** [comment]
Great thank you
